File: Lib/Telerik/Examples/QuickStart/SyntaxEditor/FirstLook/FileInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Telerik.Examples.WinControls.SyntaxEditor.FirstLook
{
    public class FileInfo
    {
        public FileInfo(string fullPath, string name)
        {
            this.FullPath = fullPath;
            this.Name = name;
        }

        public string FullPath
        {
            get;
            set;
        }

        public string Name
        {
            get;
            set;
        }
    }
}

File: Lib/Telerik/Examples/QuickStart/SyntaxEditor/FirstLook/FileInfo.vb
Namespace Telerik.Examples.WinControls.SyntaxEditor.FirstLook
    Public Class FileInfo
        Public Sub New(ByVal fullPath As String, ByVal name As String)
            Me.FullPath = fullPath
            Me.Name = name
        End Sub

        Public Property FullPath As String
        Public Property Name As String
    End Class
End Namespace